### ISSUE TITLE
[Bug]: Fee invalid prevents user from proceeding with high fee

### ISSUE THREAD

**Issue by JeanlChristophe** [synonymdev/bitkit-react-native#1704]
### Describe the bug

I want to send 60k sats via onchain.
I set the fees to 200 sat/vb so my tx will cost 41800 sats.
The application stop me from sending my tx because the fees represent half of my tx, even if I want to.

To do: it should warn them more clearly with an option to proceed .

### Screenshots / Recording

![Screenshot_20240419-113030](https://github.com/synonymdev/bitkit/assets/46056211/fa5b2592-a634-46b2-a054-be516bda440c)

### Operating system

Android, iOS

### Bitkit version

v114

**Comment by JeanlChristophe** [comment]
![Image](https://github.com/synonymdev/bitkit/assets/46056211/cc94117f-5a70-493c-be3c-4c8a12f0a217)

catch-21: The boost here is so long that it would not be useful

**Comment by coreyphillips** [comment]
This is intentional, expected behavior that's been discussed on multiple occasions and prevents users from losing sats. I would, in earnest, consider it unethical to remove that check. Even attempting to broadcast through Electrum outside of Bitkit you'll notice that the transaction will get denied. 

The Boost, however, appears be a UI/logic bug.

**Comment by JeanlChristophe** [comment]
don't think that the notice should be removed, but that once the user has seen it, he should be able to make the transaction if he wish.

**Comment by coreyphillips** [comment]
I would disagree. It's a vital check and should not be removed. Again, even if we removed this check in Bitkit most services and Electrum servers will still refuse to broadcast that transaction.

**Comment by JeanlChristophe** [comment]
Hey I talked to John and he understand the nuance. It there a way to flag the transaction on Mempool that this weird fees tx is approved by user?

**Comment by coreyphillips** [comment]
@JeanlChristophe & @BitcoinErrorLog, I think the only safe way to expose a high-fee bypass to the user in Bitkit would be to have them intentionally trigger the Send flow from the Advanced Settings in Bitkit. We could label it under a "Dangerous" sub-header as "Send High-Fee Transaction" or something similar. This would ensure it's not exposed in the normal send flow and prevent absent-minded users from accidentally triggering the bypass.

**Comment by pwltr** [comment]
There is a very basic use case that is being prevented by this limit. Let's say I want to pay for a 3$ coffee and the fee is (a rather low) 20 sat/vbyte. Bitkit prevents me from doing this tx. (Shown in the screencast by trying to set custom fee but imagine it's just the average fee rate). At the very least it should be smarter about input/output amounts and only showing the warning when the fee and or the amount is actually high in my opinion.

https://github.com/synonymdev/bitkit/assets/8538369/8f4090e6-4aca-451c-aab1-4a0a0c403c6a

**Comment by JeanlChristophe** [comment]
What do you think @BitcoinErrorLog

**Comment by coreyphillips** [comment]
We can consider removing this safety check for smaller amounts of <= 10,000 sats, but keep it in place for everything else. Only downside is we may not be able to successfully broadcast a tx with a fee greater than the outputs using our Electrum server or any other Electrum server the user connects to. So this will need to be tested.
